mrrc/cli/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Copyright (c) 2021 Red Hat, Inc
+All rights reserved.
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
```

Comment:
This license header is not correct because we're using APL v2 but not BSD license in commonjava projects.

Reply:
fixed licesing.